index.html

```diff
@@ -8216,6 +8217,107 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="suggestion">
+			<rdef>suggestion</rdef>
+			<div class="role-description">
+					<p>A single proposed change to content that can be approved or rejected.</p>
+					<p>For example, in an editing system that supports multiple authors, one author may suggest a change, and another author would be responsible
+					for accepting or rejecting the suggestion.
+					The suggested change MUST be present as child content, as an <rref>insertion</rref>, a <rref>deletion</rref>, or one of each. A suggestion MUST NOT contain
```

Comment:
In most role definitions, we present these types of structural requirements as normative author MUST statements. The spec is not consistent in this regard; for example, compare option to listbox or listitem to list. I think making them normative author MUST statements is the better editorial practice. It makes the spec more clear.

```suggestion
					Authors MUST ensure that a <code>suggestion</code> contains either one <rref>insertion</rref> child or one <rref>deletion</rref> child or ensure that it contains two children where one is an <rref>insertion</rref> and the other is a <rref>deletion</rref>.
```